cypress/e2e/pages/liveRadio/testsForCanonicalOnly.js

```diff
@@ -75,24 +67,11 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
           );
 
           if (scheduleIsEnabled) {
-            const schedulePath = Cypress.env('currentPath')
-              .replace('liveradio', 'schedule.json')
-              // the schedule call for afaanoromoo is made to bbc_oromo_radio
-              .replace('bbc_afaanoromoo_radio', 'bbc_oromo_radio');
-
-            cy.request(schedulePath).then(({ body: scheduleJson }) => {
-              const { schedules } = scheduleJson;
-              const isProgramValid = getIsProgramValid(() => {});
-              const validSchedules = schedules.filter(isProgramValid);
-
-              const isRadioScheduleDataComplete = isScheduleDataComplete({
-                schedules: validSchedules,
-              });
+            cy.getPageDataFromWindow().then(data => {
+              const { pageData } = data;
+              const schedules = pageData.radioScheduleData;
```

Comment:
So my understanding is that instead of making an actual call we're utilising the radio schedule fixture data instead? LGTM

Reply:
^^ bump on this

Reply:
We're using the radio schedule data that's in the window.SIMORGH_DATA, so yes, it's fixture data, but we use this technique (of getting the page data from the window) on canonical pages only. 

Luckily in this case, we don't really care about the content of the schedule (and how up-to-date it is), all we care about is whether there is enough schedule data to display